frontend/src/features/properties/components/forms/subforms/columns.tsx

```diff
@@ -28,7 +35,19 @@ const generateAssessedHeader = () => {
     </>
   );
 };
-
+/** used to create NetBookValue header for table with tooltip */
+const generateNetBookValueHeader = () => {
+  return (
+    <>
+      NetBookValue
```

Comment:
I think the test is failing because of this here.
```suggestion
      Net Book Value
```